go/registry/api/api.go

```diff
@@ -1501,6 +1508,9 @@ const (
 	GasOpUpdateKeyManager transaction.Op = "update_keymanager"
 	// GasOpProveFreshness is the gas operation identifier for freshness proofs.
 	GasOpProveFreshness transaction.Op = "prove_freshness"
+	// GasOpPublishEphemeralSecret is the gas operation identifier for publishing
+	// key manager ephemeral secret.
+	GasOpPublishEphemeralSecret transaction.Op = "publish_ephemeral_secret"
```

Comment:
To be consistent with `GasOpUpdateKeyManager`. Should I move both of them to the key manager api, and introduce consensus parameters for the key manager (for default prices)?

Reply:
Oh I see, this is actually wrong, not sure why it was there, maybe as a shortcut because the key manager didn't have any gas costs defined. It would be ideal to move it, yeah. We just must make sure this is handled properly (e.g. in fix genesis).